### ISSUE TITLE
Using pretrained when training

### ISSUE THREAD

**Issue by huytuong010101** [mjkwon2021/CAT-Net#4]
According to your paper, you initialized RGB Stream weight by pretrained on ImageNet and DCT Stream by double JPEG classification. Do you think that I can train CAT-NET model without using pretrained? I reimplementing your model in tensorflow and it so difficult to load the weight of the pretrained model.
Thank you <3

**Comment by dan326326** [comment]
> Wow thank you for the helpful information <3
Hello, could you please tell me about the CASIA data set you used? Is there any change to the data set? I keep reporting errors and can't find the file, but the file path is correct

**Comment by huytuong010101** [comment]
> > Wow thank you for the helpful information <3
> > Hello, could you please tell me about the CASIA data set you used? Is there any change to the data set? I keep reporting errors and can't find the file, but the file path is correct

I am just training with small dataset to check if model is reimplemented correctly. I use `CASIAv2/Sp` for training and the loss reduce to 0.24 in first 4 epoch.

**Comment by dan326326** [comment]
hello，Where did you download your CASIAv2 dataset




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mjkwon2021/CAT-Net"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月11日(星期四) 中午12:29
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [mjkwon2021/CAT-Net] Using pretrained when training (Issue #4)





   
Wow thank you for the helpful information <3
 Hello, could you please tell me about the CASIA data set you used? Is there any change to the data set? I keep reporting errors and can't find the file, but the file path is correct
   
I am just training with small dataset to check if model is reimplemented correctly. I use CASIAv2/Sp for training and the loss reduce to 0.24 in first 4 epoch.
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by huytuong010101** [comment]
> hello，Where did you download your CASIAv2 dataset
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "mjkwon2021/CAT-Net" ***@***.***&gt;; 发送时间:&nbsp;2021年11月11日(星期四) 中午12:29 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [mjkwon2021/CAT-Net] Using pretrained when training (Issue #4) Wow thank you for the helpful information <3 Hello, could you please tell me about the CASIA data set you used? Is there any change to the data set? I keep reporting errors and can't find the file, but the file path is correct I am just training with small dataset to check if model is reimplemented correctly. I use CASIAv2/Sp for training and the loss reduce to 0.24 in first 4 epoch. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

`https://www.kaggle.com/sophatvathana/casia-dataset`

**Comment by dan326326** [comment]
really thank you!!!!!!!!!!!!!!!!!!!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mjkwon2021/CAT-Net"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月11日(星期四) 下午2:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [mjkwon2021/CAT-Net] Using pretrained when training (Issue #4)





  
hello，Where did you download your CASIAv2 dataset
 …
 ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "mjkwon2021/CAT-Net" @.&gt;; 发送时间:&nbsp;2021年11月11日(星期四) 中午12:29 @.&gt;; @.@.&gt;; 主题:&nbsp;Re: [mjkwon2021/CAT-Net] Using pretrained when training (Issue #4) Wow thank you for the helpful information <3 Hello, could you please tell me about the CASIA data set you used? Is there any change to the data set? I keep reporting errors and can't find the file, but the file path is correct I am just training with small dataset to check if model is reimplemented correctly. I use CASIAv2/Sp for training and the loss reduce to 0.24 in first 4 epoch. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.
  
https://www.kaggle.com/sophatvathana/casia-dataset
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
